envoy.dependency.check/envoy/dependency/check/abstract/checker.py

```diff
@@ -63,7 +63,7 @@ def dependencies(self) -> Tuple["abstract.ADependency", ...]:
         """Tuple of dependencies."""
         deps = []
         for k, v in self.dependency_metadata.items():
-            deps.append(self.dependency_class(k, v, self.github))
+            deps.append(self.dependency_class(k, v, self.github, self.session))
```

Comment:
`self.github` has `self.github.session` - i think rather just reuse the session from the github object